pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLPdpAlbum.java

```diff
@@ -52,27 +58,61 @@ public boolean loadNextPageOfPhotos(final RequestHandlers handlers) {
                 return false;
             }
             PXLClient pxlClient = PXLClient.getInstance(context);
-            String requestPath = "albums/from_sku";
             this.pagesLoading.put(desiredPage, true);
             this.handlers = handlers;
-            pxlClient.makeCall(requestPath, getRequestParams(desiredPage), this);
+
+            try {
+                pxlClient
+                        .getBasicrepo()
+                        .getPhotosWithSKU(
+                                sku,
+                                PXLClient.apiKey,
+                                filterOptions != null ? filterOptions.toParamString() : null,
+                                sortOptions != null ? sortOptions.toParamString() : null,
+                                perPage,
+                                desiredPage
+                        ).enqueue(
+                        new Callback<String>() {
+                            @Override
+                            public void onResponse(Call<String> call, Response<String> response) {
+                                try {
+                                    String result = response.body();
+                                    Log.e("retrofit result","retrofit result:" + result);
```

Comment:
should we just make "retrofit result" a global log tag that we can use everywhere through out the sdk?